data/brands/amenity/bank.json

```diff
@@ -11196,15 +11213,16 @@
       "displayName": "하나은행",
       "id": "koreaexchangebank-10d607",
       "locationSet": {"include": ["kr"]},
+      "matchNames": ["KEB하나은행"],
       "tags": {
         "amenity": "bank",
         "brand": "하나은행",
-        "brand:en": "Korea Exchange Bank",
+        "brand:en": "KEB Hana Bank",
```

Comment:
According to the [English Wikipedia article](https://en.wikipedia.org/wiki/Hana_Bank), it looks like they're branded in South Korea as just "Hana Bank" in English. Let me know if this is not the case

Reply:
In general, it is called "Hana Bank", but they have designated the official English name, "KEB Hana Bank".